.travis.yml

```diff
@@ -31,6 +30,7 @@ before_install:
   - composer self-update
   - if [ "$SYMFONY_VERSION" != "" ]; then composer require --dev --no-update symfony/symfony=$SYMFONY_VERSION; fi
 
-install: composer update $COMPOSER_FLAGS --prefer-source
+install: composer update --prefer-dist --no-interaction $COMPOSER_FLAGS
 
-script: ./vendor/bin/phpunit
+script:
+  - phpunit --coverage-text
```

Comment:
Coverage won't work, as xdebug [is disabled](#diff-354f30a63fb0907d4ad57269548329e3L29).